### DISCUSSION TITLE
33.1.1.533_BETA Release Notes

### DISCUSSION THREAD

**Discussion by urthling** [airsdk/Adobe-Runtime-Support#912] [Q&A]
Can we see what the latest build has - I'd love to take a look but not sure what I'm looking at without Release Notes

**Comment by ajwfrost** [comment]
Looks like the links don't quite work properly when we throw in some text as well as a version number.. but you can get to the document at
https://airsdk.harman.com/api/versions/33.1.1.533/release-notes/Release_Notes_AIR_SDK_33.1.1.533.pdf
and we'll try to upload the summary into the database shortly so you see it also online.

Mostly:
Android API 30 -> proper App Bundle support plus the other things that Google required here
iOS -> Further code-signature fixes (hopefully that's sorted all of the outstanding issues now...)
Media -> various fixes and updates on Windows
Plus a fair amount of other minor bug-fixes...

**Reply by Adolio** [reply]
"...work has now started on revising the multimedia capabilities." 🤩

**Comment by leossmith** [comment]
Is there way to tell adt.cfg to export to aab?

**Reply by ajwfrost** [reply]
Yes, if you're using an IDE that doesn't have this option then try to generate an APK file but uncomment the below in the adt.cfg file:
```
#CreateAndroidAppBundle=true
```
Although depending on the options provided to the command-line, you may end up with a file called something.apk that's actually an app bundle. If the command-line is given "output_package" then it would turn into "output_package.aab" but if it's given "output_package.apk" then that's what we use....

The other thing to watch for is that you need the -platformsdk option to be provided for Android SDK so that we can build the bundle properly; plus it needs a JAVA_HOME environment variable for a JDK (simplest perhaps to use the one from Android Studio?).

**Reply by matrix3d** [reply]
how cmd build linux?

**Reply by ajwfrost** [reply]
For Linux, I'm waiting for the website to get updated, we're a little hit by covid-related delays this week :-(

**Reply by ajwfrost** [reply]
@matrix3d the website is updated now, so if you refresh it you can download the Linux SDKs....

**Reply by matrix3d** [reply]
> @matrix3d the website is updated now, so if you refresh it you can download the Linux SDKs....

```
 ./adt -package -tsa none -storetype pkcs12 -keystore "bat/tools3d.p12" -storepass fd -target bundle l_tools3d_bundle application33.xml
File /home/li/桌面/airsdk/bin/application33.xml is not a valid AIRI or AIR file

```

**Reply by matrix3d** [reply]
> @matrix3d the website is updated now, so if you refresh it you can download the Linux SDKs....

/mnt/d/sdk/AIRSDK_Linux_33.1.1.533/bin/adt -package -tsa none -storetype pkcs12 -keystore "bat/tools3d.p12" -storepass fd -target bundle air/l_tools3d_bundle application33.xml
-C bin .
Failed to load the AIR Runtime
